src/WebJobs.Script/functionTemplate.js

```diff
@@ -1,9 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-var f = require('{0}');
+var f = require('{0}'),
+    util = require('util');
 
 return function (context, callback) {{
+    var origLog = context.log;
+    context.log = function(value) {{
+        origLog(util.inspect(value));
```

Comment:
`util.format` works great, thanks. `context.log` now supports more of what users expect, e.g. `context.log('Node.js queue trigger function processed work item %s', workItem.id);`. I updated all the samples in the script repo to take advantage of this. We should probably update the templates too.